openbb_terminal/miscellaneous/integration_tests_scripts/fixedincome/test_fixedincome.openbb

```diff
@@ -12,12 +12,12 @@ sofr --help
 sonia -s 2022-01-01 -e 2023-01-01
 sonia --parameter total_nominal_value
 sonia --help
-ameribor -s 2022-01-01 -e 2023-01-01 --parameter 90_day_average
+ameribor -s 2022-01-01 -e 2023-01-01 --parameter 90_day_ma
 ameribor --help
 fed -s 2022-01-01 -e 2023-01-01
 fed --parameter volume --target --quantiles
 fed --help
-iorb -s 2022-01-01 -e 2023-01-01 --export
+iorb -s 2022-01-01 -e 2023-01-01 --export xlsx --sheet-name IORB
```

Comment:
Thinking about all the integration tests, I'm scared to look at the exports on the computers lol

Reply:
I was having this same thought, only for the unit testing that captures the HTTP response and the parsed content, so it could really start to bloat the repo with test cassettes.

Reply:
Probably.  Or at least to run it weekly or something.  These things have been running daily (or more) for months haha

Reply:
rip